spec/features/dashboard_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.feature "Filter content items", type: :feature do
+  scenario "the user selects an organisation from the organisations select box, clicks the filter button and retrieves a filtered list of the organisation's content items" do
+    create :organisation, slug: "the-slug-1", title: "title 1"
+    create :organisation, slug: "the-slug-2", title: "title 2"
+
+    expected_path = URI.escape "/content_items?utf8=✓&organisation_slug=the-slug-2"
+
+    visit "/"
```

Comment:
I would visit the URL that has the filter, not the root of the application. What do you think?